cns/client.go

```diff
@@ -282,3 +287,19 @@ func (c *Client) ReconfigVolumePolicy(ctx context.Context, PolicyReconfigSpecs [
 	}
 	return object.NewTask(c.vim25Client, res.Returnval), nil
 }
+
+// SyncDatastore calls the CnsSyncDatastore API
+func (c *Client) SyncDatastore(ctx context.Context, dsURL string, fullSync ...bool) (*object.Task, error) {
+	req := cnstypes.CnsSyncDatastore{
+		This:         CnsDebugManagerInstance,
+		DatastoreUrl: dsURL,
+	}
+	if len(fullSync) != 0 {
```

Comment:
change this to 

```
func (c *Client) SyncDatastore(ctx context.Context, dsURL string, fullSync bool) (*object.Task, error) {
```

Reply:
This is an optional field, so not adding it always.

Reply:
dsURL is also optional